doc/source/news.rst

```diff
@@ -5,6 +5,48 @@
 News
 ====
 
+.. _release-12-0-9:
+
+Release 12.0.9 - 2022-10-28
+---------------------------
+
+Improvements
+------------
+
+* [:doc:`reference/functions/escalate`] Added a document for the ``escalate()`` function.
+
+* [:doc:`reference/normalizers`] Added NormalizerHTML. (Experimental)
+
+  This is a normalizer for HTML.
+
+  Currently this supports only unescaping reference characters like ``&amp;`` or ``&#38;``.
+
+  Here is a sample query for NormalizerHTML.
+
+  .. code-block::
+  
+     normalize NormalizerHTML "Groonga &amp; Mroonga &#38; Rroonga"
+     [[0,1666854852.063215,0.0006151199340820312],{"normalized":"Groonga & Mroonga & Rroonga","types":[],"checks":[]}]
+  
+  In this sample, ``&amp;`` and ``&#38;`` are unescaped to ``&``.
+
+* [httpd] Updated bundled nginx to 1.23.2.
```

Comment:
This update include security bug fix.
Therefore, we should also add about security bug fix.

See: https://nginx.org/en/CHANGES